spec/features/idv/phone_otp_rate_limiting_spec.rb

```diff
@@ -37,7 +37,7 @@
       complete_idv_steps_before_phone_otp_verification_step(user)
 
       max_attempts.times do
-        fill_in('code', with: 'wrong')
+        fill_in('code', with: 'badbad')
```

Comment:
Good example of an improvement: Previously the user would have been able to submit a code in an invalid format such as this (and risk getting locked out because of it! the whole point of this spec). But now we validate client-side.